app/controllers/municipality_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MunicipalityController < ApplicationController
+  # GET '/municipalities'
+  def index
+    render json: Municipality.all
+  end
+
+  # GET '/municipality/:id'
+  def find_by_id
```

Comment:
In rails this action is the `show`, just to follow the convention, please change this

Reply:
Done!